config/routes.rb

```diff
@@ -13,6 +13,8 @@
     end
 
     post :create
+    delete :inactviate
```

Comment:
Is this thing used at all? If not, then let's delete it. If it is, then let's fix the typo (`v` and `i` are in the wrong places), but even better, can we use `deactivate` as the verb instead, which is think is more [grammatically correct](https://english.stackexchange.com/questions/312339/inactive-or-deactivated/312346).